apis/core/bootstrap/shapes/dataset.ts

```diff
@@ -46,6 +51,40 @@ const euThemesQuery = sparql`construct {
   }
 }`
 
+const fileTypesQuery = sparql`construct {
+  ?c a ${hydra.Collection} .
+  ?c ${hydra.member} ?term .
+  ?term ${rdfs.label} ?name .
+} WHERE {
+  BIND ( bnode('collection') as ?c )
+
+  graph <https://lindas.admin.ch/ontologies> {
+    ?term a ${euvoc.FileType} ;
+     ${skos.prefLabel} ?name ;
+      ${skos.inScheme} <http://publications.europa.eu/resource/authority/file-type> .
+
+    FILTER(regex(?name, "_q_", "i"))
```

Comment:
why the filter? in TEST,it filters out all values

Reply:
That's a template. Does `_q_` instead of `{q}` to prevent URL encoding an gets replaced later.

From the UI, the value of `q` is provided from the auto-complete editor

Reply:
Yes. The queries go against `PUBLIC_QUERY_ENDPOINT`.

You can add `PUBLIC_QUERY_ENDPOINT=https://test.lindas.admin.ch/query` to `.env` in project root and then `lando rebuild`